### PR TITLE
Added note about redirecting on specific URL after flash message

### PR DESCRIPTION
**Pull request by Machy8**
- Updated components.texy (both languages)
- Added note about redirecting on specific url after flash message

### CODE REVIEW

**Comment by dg** [comment]
Why do you use redirectUrl instead of redirect? `redirect()` should handle _fid automatically.

File: cs/components.texy [hunk 1]
```diff
@@ -108,7 +108,28 @@ Příklad:
...
+    // ... požádáme model o smazání záznamu ...
+
+    $this->presenter->flashMessage('Položka byla smazána.');
+
+    if ($this->presenter->hasFlashSession()) {
+        $url = new Url($url);
+        $url->appendQuery([Presenter::FLASH_KEY => $this->presenter->getParameter(Presenter::FLASH_KEY)]);
+    }
+
+    $this->presenter->redirectUrl($url);
```
Comment (f3l1x) [inline cs/components.texy]: Ted jsem z toho uplnej jelen, proc je tam potreba ten fid? :-)

Comment (f3l1x) [inline cs/components.texy]: Neni to @Machy8 nejak moc komplikovany? Co jsi tim vlastne myslel?

**Comment by Machy8** [comment]
I have been working on redirecting on the same page after user signed in. (Product detail => page with login form => Product detail) and in the first steps I have used something like the following code (simplified).

````PHP
/**
 * @persistent
 * @var string
 */
public $backlink;
...

$backlink = $this->presenter->getParameter('backlink');
...

if ($backlink && $url = $this->sessionStorage->remove($backlink)) {
 if ($this->presenter->hasFlashSession()) {
    $url = new Url($url);
    $url->appendQuery([Presenter::FLASH_KEY => $this->presenter->getParameter(Presenter::FLASH_KEY)]);
  }
  $this->presenter->redirectUrl($url);
}
````


Then I have found https://pla.nette.org/en/redirect-to-same-page-after-form-submit so I rewrote the awkward solution into a better one. Anyway it took me a while to find out how to keep flash messages shown after redirecting on some specific URL so I though this could be useful for someone and I decided to send this pull request with the info into the docs. If you see this as unnecessary, close it :)

**Comment by dg** [comment]
This should not be in the documentation. If this is a framework issue, it should be fixed, if it is a workaround, it should appear somewhere else.

**Comment by f3l1x** [comment]
@dg I'm glad you say that, @Machy8 what about add this to new planette? https://github.com/planette/planette?

**Comment by Machy8** [comment]
@dg Understand. 
@f3l1x Good Idea, I will write some short text about that on weekend.

**Comment by f3l1x** [comment]
If you don't mind, I am gonna close this PR. Please open it on planette, I'm looking forward. Thank you.

**Pull request closed without merging**